librz/util/calc.c

```diff
@@ -186,6 +186,23 @@ static RzNumCalcValue term(RzNum *num, RzNumCalc *nc, int get) {
 				return d;
 			}
 			left = Ndiv(left, d);
+		} else if (nc->curr_tok == RNCEXP) {
```

Comment:
Whether to place this here or in `expr` or in `term` depends on which precedence we want to give to the exponential operation.

Could you put some tests with more complicated expressions so we can check it works as we'd want?

In python, the following is true:
```
>>> 10 ** 2 * 3
300
>>> 10 ** 2 / 3
33.333333333333336
>>> 10 ** 2 + 3
103
>>> 
```